mock/mockSyfotilgangskontroll.ts

```diff
@@ -1,4 +1,4 @@
-export {};
+const auth = require("../server/auth");
```

Comment:
Kan man ikke bruke `import { ensureAuthenticated } from "../server/auth";` her?

Reply:
Ja eller nei: Jeg kåla med de greiene der. 
Det krevever endel endringer config for å kunne bruke ecmascript i Nodejs, slik at det bundles riktig.
Man kan bruke det for lokal kjøring, men ble mye styr for å få til det pluss alt andre, så utsatte et. :/

Reply:
Den er grei 👍